cvat/apps/engine/views.py

```diff
@@ -1867,6 +1869,121 @@ def get_serializer_class(self):
     def perform_create(self, serializer, **kwargs):
         super().perform_create(serializer, owner=self.request.user)
 
+
+@extend_schema(tags=['labels'])
+@extend_schema_view(
+    retrieve=extend_schema(
+        summary='Method returns details of an label',
+        responses={
+            '200': LabelSerializer,
+        }),
+    list=extend_schema(
+        summary='Method returns a paginated list of labels',
+        parameters=[
+            # These filters are implemented differently from others
+            OpenApiParameter('job_id', description='A simple equality filter for job id'),
+            OpenApiParameter('task_id', description='A simple equality filter for task id'),
+        ],
+        responses={
+            '200': LabelSerializer(many=True),
+        }),
+    partial_update=extend_schema(
+        summary='Methods does a partial update of chosen fields in an label',
```

Comment:
I guess it would be better to add information about how this endpoint can be used to delete label (when `deleted` is specified)

Reply:
I've added it to the field description.